static/components/around-cards.js

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+// Component
+import PoiCard from './PoiCard'
+
+class App extends React.Component {
+
+  constructor(props) {
+      super(props)
+      this.state = {
+        pois: {},
+        userView: {},
+        userLocation: {},
+        mapBounds: {}
+      }
+    }
+
+  componentDidMount() {
+    fetch('/around/spots.json')
+      .then(response => response.json())
+      .then(response => {
+        this.setState({
+          pois: response
+        })
+      })
+    map.on('moveend', move => {
+      const centerMap = map.getCenter()
+      const bounds = map.getBounds()
+      this.setState({
+        userView: centerMap,
+        mapBounds: bounds
+      })
+    })
+    navigator.geolocation.getCurrentPosition(position => {
+      this.setState({
+        userLocation: [position.coords.latitude, position.coords.longitude]
+      })
+    })
+  }
+
+  render() {
+    const cards = Object
+      .keys(this.state.pois)
+      .filter(key => {
+        return (
+          Math.round(this.state.pois[key].position.latitude) == Math.round(this.state.userView.lat)
```

Comment:
user triple equals (_===_) whenever possible.
You should be able to use it here and the line below.